src/commands/auth/delete.ts

```diff
@@ -42,7 +42,7 @@ export default class Delete extends Command {
 
   async run() {
     const { args, flags } = this.parse(Delete)
-    await ChectlContext.init(flags, this)
+    await ChectlContext.initAndGet(flags, this)
```

Comment:
For this and all identical changes: what is the reason for replacing `init` with `initAndGet` ?

Reply:
My mistake